docs/nitclk/class.rst

```diff
@@ -1,442 +1,445 @@
-.. py:module:: nitclk
-
-Session
-=======
-
-.. py:class:: Session(self)
-
-    
-
-    TBD
 
-    
+.. py:module:: nitclk
 
+Public API
+==========
 
+NI-TClk python module consists of several functions and properties. The functions are not sessions based
+like other nimi-python driver support and are called directly on the nitclk module. See :py:class:`SessionReference`
+for properties.
 
+.. code:: python
 
-Methods
-=======
+    with niscope.Session('dev1') as scope1, niscope.Session('dev2') as scope2:
+        nitclk.initiate([scope1, scope2])
```

Comment:
you have to call configure_for_homogeneous_triggers

Reply:
Added